CHANGELOG.rst

```diff
@@ -5,11 +5,13 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
-0.3.0-SNAPSHOT (2021-07-20)
+0.3.0-SNAPSHOT (2021-08-18)
```

Comment:
Nit: The date should be updated to the date of this merge

Reply:
I don't think we need to do it here since we will do it in the release PR anyhow :D

Reply:
True, that's why i set it to Nit 👍